File: Medium/2025-08-29-3021-Alice-and-Bob-Playing-Flower-Game/solution.java
java
class Solution {
    public int countPairs(int n, int m) {
        int count = 0;
        for (int x = 1; x <= n; x++) {
            for (int y = 1; y <= m; y++) {
                if ((x + y) % 2 == 1) {
                    count++;
                }
            }
        }
        return count;
    }
}

File: Medium/2025-08-29-3021-Alice-and-Bob-Playing-Flower-Game/solution.py
python
class Solution:
    def countPairs(self, n: int, m: int) -> int:
        count = 0
        for x in range(1, n + 1):
            for y in range(1, m + 1):
                if (x + y) % 2 == 1:
                    count += 1
        return count